Add snmp_tile template to ddev create --type (#8216)
DataDog/integrations-core: datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/snmp_tile/{check_name}/README.md

# Agent Check: snmp_{integration_name}

## Overview

This check monitors [{integration_name}][1].

> For details of monitored metrics see the [SNMP integration tile][2]

Reviewer: ```suggestion
For details of monitored metrics see the [Network Device Monitoring][2] documentation.
```

## Setup

### Installation

{install_info}

### Configuration

1. <List of steps to setup this Integration>

### Validation

> <Steps to validate integration is functioning as expected>

Reviewer 2: Are those sections needed ?

Author: removed

## Data Collected

### Metrics

> {integration_name} does not include any metrics.

Reviewer: ```suggestion
The {integration_name} integration does not include any metrics.
```

> The {integration_name} integration does not include any metrics.

Reviewer 2: > The {integration_name} integration does not include any metrics.

I think all snmp vendor tiles will have metrics, so, this ^ might be a bit misleading.

Author: This should be `For details of monitored metrics see the [SNMP integration tile][2]`

### Service Checks

> {integration_name} does not include any service checks.

Reviewer: ```suggestion
The {integration_name} integration does not include any service checks.
```

### Events

> {integration_name} does not include any events.

Reviewer: ```suggestion
The {integration_name} integration does not include any events.
```

## Troubleshooting

Need help? Contact [Datadog support][1].

[1]: https://docs.datadoghq.com/help/

> [2]: https://app.datadoghq.com/account/settings#integrations/snmp

Reviewer: ```suggestion
[2]: https://docs.datadoghq.com/network_performance_monitoring/devices/data
```

Author: > Docs review

Thank you @ruthnaebeck, committed your suggestions, should be good now. Can you give it another review round please?
